src/main/webapp/components/basic-search/basic-search.js

```diff
@@ -266,7 +266,7 @@ const createQuery = filterMap => {
 
   return {
     filterTree: toFilterTree(filterMap),
-    sources: sources || null,
+    sourceIds: sources || null,
```

Comment:
This might break searches using the basic search form in workspaces https://github.com/connexta/revelio/blob/master/src/main/webapp/components/workspace/workspace.js#L65. Could you instead transform the sources property for the query in simple search?